include/boot/CAN.hpp

```diff
@@ -0,0 +1,17 @@
+#ifndef EVT_BOOTCAN_HPP
+#define EVT_BOOTCAN_HPP
+
+#include <boot/IOInterface.hpp>
+
+class CAN : public IOInterface {
```

Comment:
Some docs explaining how this is fundamentally different from the other `CAN.hpp` file that we have would be nice.